include/ex_actor/internal/actor_creation.h

```diff
@@ -264,14 +270,27 @@ class ActorRegistry {
   void HandleNetworkRequest(uint64_t receive_request_id, network::ByteBufferType request_buffer) {
     serde::BufferReader<network::ByteBufferType> reader(std::move(request_buffer));
     auto message_type = reader.NextPrimitive<serde::NetworkRequestType>();
+
     if (message_type == serde::NetworkRequestType::kActorCreationRequest) {
       auto actor_cls_index = reader.NextPrimitive<uint64_t>();
-      uint64_t actor_id = FindClassAndCreateActor(actor_cls_index, reader.Current(), reader.RemainingSize());
-      serde::BufferWriter<network::ByteBufferType> writer(network::ByteBufferType(sizeof(actor_id)));
-      writer.WritePrimitive(actor_id);
-      message_broker_->ReplyRequest(receive_request_id, std::move(writer).MoveBufferOut());
+      try {
+        uint64_t actor_id = FindClassAndCreateActor(actor_cls_index, reader.Current(), reader.RemainingSize());
+        serde::BufferWriter<network::ByteBufferType> writer(
+            network::ByteBufferType(sizeof(serde::NetworkReplyType) + sizeof(actor_id)));
+        writer.WritePrimitive(serde::NetworkReplyType::kActorCreationReturn);
+        writer.WritePrimitive(actor_id);
+        message_broker_->ReplyRequest(receive_request_id, std::move(writer).MoveBufferOut());
+      } catch (std::exception& error) {
+        std::vector<char> serialized = serde::Serialize(serde::ActorMethodReturnValue {std::string(error.what())});
```

Comment:
BTW, can you also add the exception's type to the error msg? can get from `typeid(exception).name()` IIRC. The same for actor method call's exception.